html-relative-links/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+  <li><a href="./index.html">Home Page</a></li>
+  <li><a href="../html-relative-links/about-me/about-me.html">About Me</a></li>
+  <li><a href="../html-relative-links/projects/projects.html">Projects</a></li>
+  <li><a href="../html-relative-links/contact-us/contact-us.html">Contact Us</a></li>
```

Comment:
You don't need to use `..` here. these folders are in the same directory that `index.html` is in.  So `about-me/about-me.html` works fine.